projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts

```diff
@@ -231,6 +231,14 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
       value: 'Resolve Issues',
       type: TabbingOrderTypes.LINK,
     },
+    {
```

Comment:
[product-configurator-tabbing.core-e2e-spec-flaky.ts] fails with standard backend https://40.76.109.9:9002". Our tests should be able to run with different backend versions, correct? TBD how we achieve that, in other suites we introduced a backend release setting

Reply:
Done!